constants/members.json

```diff
@@ -3429,6 +3441,76 @@
       "term": 1235,
       "teams": ["shw"],
       "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Fdefault.png?alt=media&token=fe95cc90-ba2b-4c04-a808-0f903cc8b519"
+    },
+    {
+      "name": "Niharika Srivatsa",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["haven"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FDev_Niharika_Srivatsa.jpg?alt=media&token=38202775-87fa-4078-8c6a-2aa7fa7bd1c9"
+    },
+    {
+      "name": "Jesse Li",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["internal-tools"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FDev_Jesse_Li.jpg?alt=media&token=c4ffd1f3-ef13-4683-87b3-aa7c7ec8006a"
+    },
+    {
+      "name": "Maggie Lu",
+      "role": "vp engineering",
+      "term": 1236,
+      "teams": ["executive"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FVPE_Maggie_Lu.JPG?alt=media&token=ebc4f915-0d39-4934-a3b5-fea3c55a31d3"
+    },
+    {
+      "name": "Annie Cai",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["internal-tools"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2Fdev_annie_cai.jpg?alt=media&token=63b6e3b5-35a7-4151-b7fd-7ebb11abfb84"
+    },
+    {
+      "name": "Maisha Thasin",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["infra"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2Fdev_maisha_thasin.jpeg?alt=media&token=b4ddba89-ccd2-4345-b136-5b80475ca4a7"
+    },
+    {
+      "name": "Maisha Thasin",
+      "role": "developer",
+      "term": 1236,
+      "teams": ["mp"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FPL_Shahan_Nedadahandeh.png?alt=media&token=13fe9646-4c81-494c-a81f-b0b2aa8ef396"
+    },
+    {
+      "name": "Parth Desai",
+      "role": "vp scoping",
+      "term": 1236,
+      "teams": ["executive"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FVPS_Parth_Desai.JPG?alt=media&token=7e9af2e8-e8e0-4d51-94e8-dbcf9ef4124c"
+    },
+    {
+      "name": "Janaki Menon",
+      "role": "vp talent",
+      "term": 1236,
+      "teams": ["executive"],
+      "img": "https://firebasestorage.googleapis.com/v0/b/uw-blueprint.appspot.com/o/img%2Ff23headshosts%2FVP%20Talent_Janaki_Menon.jpeg?alt=media&token=51ef769e-0970-44d5-b0ef-7051f920cddc"
+    },
+    {
+      "name": "Janaki Menon",
```

Comment:
Same here, this should be for Vaaranan instead of Janaki

Reply:
@HeetShah can you help confirm?

Reply:
Yes, the `img` link is for `Vaaranan Yogalingam `. `Janaki` is the one above which you have right